### PR TITLE
Added Card Component & fixed missing logo in footer

### LINKED ISSUE #40: [Site] Why getNightHawk (card component)
**Current State of the GetNighthawk.dev site**
There is a "Why Getnighthawk section already present on the current website

**Desired State of the GetNighthawk.dev site**
screenshot from Figma:
chrome_OnTKAhdxeI
This will require the development of the card component with a transition effect.
---
**Contributor Resources**
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md.
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by chinmaym07**
Signed-off-by: chinmaym07 <b418020@iiit-bh.ac.in>

**Description**

This PR fixes #40 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/_sass/getnighthawk.scss [hunk 1]
```diff
@@ -5445,5 +5445,113 @@ a.globalFooterCard.card-environment, div.globalFooterCard.card-environment {
...
+            h4{
+                font-weight: bolder;
+            }   
+        }
+    }
+
+    .card-item:after {
+        display:block;
+        content: '';
+        border-bottom: 0.3rem solid $saffron;
```
Comment (vinayaksh42) [inline docs/_sass/getnighthawk.scss:5492]: change `0.3rem` to `0.4rem`

File: docs/_sass/getnighthawk.scss [hunk 2]
```diff
@@ -5445,5 +5445,113 @@ a.globalFooterCard.card-environment, div.globalFooterCard.card-environment {
...
+            }
+        }
+        .card-cust-body{
+            padding: 1rem 0rem;
+            margin-top: auto;
+            h4 , p{
+                padding: 0.5rem 1rem;
+            }
+            h4{
+                font-weight: bolder;
```
Comment (vinayaksh42) [inline docs/_sass/getnighthawk.scss:5484]: Increase the font-weight to `900`

File: docs/_sass/getnighthawk.scss [hunk 3]
```diff
@@ -5445,5 +5445,113 @@ a.globalFooterCard.card-environment, div.globalFooterCard.card-environment {
...
+        display:block;
+        content: '';
+        border-bottom: 0.3rem solid $saffron;
+        transform: scaleX(0);  
+        transition: transform 350ms ease-in-out;
+        width: 100%;
+        margin-top: auto;
+    }
+    
+    .card-item:hover:after {
```
Comment (vinayaksh42) [inline docs/_sass/getnighthawk.scss:5499]: add another case separately for .card-item:hover, which should include the increased box-shadow.

File: docs/_sass/getnighthawk.scss [hunk 4]
```diff
@@ -5445,5 +5445,113 @@ a.globalFooterCard.card-environment, div.globalFooterCard.card-environment {
...
+    .card-item{
+        text-align: center;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        border-radius:5px;
+        margin:0rem 1rem;
+        height: 22rem;
+        box-shadow:0 2px 10px rgba(0,0,0,0.4);
```
Comment (vinayaksh42) [inline docs/_sass/getnighthawk.scss:5470]: Decrease the `box-shadow` to bare minimum
chrome_rNGOhrTFG6

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]

**Comment by vinayaksh42** [comment]
Initially the card should have a very faint shadow and upon hovering it should increase, also increase the height of the yellow line by 2 times.
chrome_JhAqzdfG6f

File: docs/_sass/getnighthawk.scss (hunk 2 above)
Comment (leecalcote) [reply docs/_sass/getnighthawk.scss:5484]:
```suggestion
                font-weight: 900;

```

File: docs/_sass/getnighthawk.scss (hunk 1 above)
Comment (leecalcote) [reply docs/_sass/getnighthawk.scss:5492]:
```suggestion
        border-bottom: 0.4rem solid $saffron;

```

**Comment by leecalcote** [comment]
Nice!

**Review by vinayaksh42 — APPROVED** [review]
Looks great now!

**Review by leecalcote — APPROVED** [review]

**Pull request merged**